examples/complete/main.tf

```diff
@@ -12,13 +12,56 @@ module "rg" {
   groups = {
     demo = {
       name   = module.naming.resource_group.name
-      region = "northeurope"
+      region = "germanywestcentral"
     }
   }
 }
 
 module "cosmosdb" {
   source = "../../"
```

Comment:
Shouldn't this be set to the github repo with a version?

Reply:
solved